script/Ciel/Day 12/12_01_CIEL12_3_2A.txt

```diff
@@ -131,102 +143,108 @@ I feel like I saw traces of fresh blood on the corridor walls.
+ If I do, it'll all just work itself out, same as it did this morning.
 }
 [sha:466e63215a28df14bed521f5e5ae56491db9c3a8]{
 -- Page 189, Offset 31377.
 -- いつの間にか日は沈んでいた。
-Before I know it, the sun has set.
+Before I know it, the sun has already set.
 }
 [sha:19b7820cb637c8c8650c815a2f1566482d8c000f]{
 -- Page 189, Offset 31378.
 -- 琥珀の用意した薬は気休め程度には効果があった。
-The medicine that Kohaku prepared helped bring me peace of mind.
+// No -さん here
+The medicine Kohaku prepared managed to calm me down, if nothing else.
 }
 [sha:c7608fde225fc374682f4169c0115e9da85f1383]{
 -- Page 189, Offset 31379.
 -- 頭痛は治まらないものの、手足の鈍さ、かゆさは気にならなくなった。
-Although my headache hasn't subsided yet, the numbness and itchiness in my limbs are no longer a problem.
+My headache hasn't gone away,
+ but at least my limbs no longer feel numb and itchy.
 }
 [sha:139c57d600330d1f9d656a92749d806747bccf1f]{
 -- Page 189, Offset 31380.
 -- 俺はベッドに横になっている。
-I'm lying down on the bed.
+I'm lying in bed.
 }
 [sha:7d62211f95be64a4e834d0ef2de2e46aa6d3918b]{
 -- Page 189, Offset 31381.
 -- 横になって、屋敷の物音に気を配っている。
-As I lie down, I pay close attention to the sounds permeating the mansion.
+// None of the following sounds are 屋敷内, so not really sounds 'of' the mansion here
+As I lie here,
+ I listen to the sounds that make their way to the mansion.
 }
 [sha:61705df94bbc9e516a482a86dd632e8c94ba0f84]{
 -- Page 190, Offset 31382.
 -- 外からは遠い喧騒が聞こえる。
-I can hear the distant clatter from outside.
+I can hear the far-off din coming from outside.
 }
 [sha:46454b56edb58bca8e13b44bab0d0de2ab6f19db]{
 -- Page 190, Offset 31383.
 -- 坂の下、屋敷から１キロ以上離れた駅前の雑踏。
-The busy crowd in front of the station below the hill, more than a kilometer away from the mansion.
+There's the sound of the crowd in front of the station down the hill,
+ over a kilometer from the mansion.
 }
 [sha:e3d7ac6c679e09e0d29f0c1c6da8f70a4ec44926]{
 -- Page 190, Offset 31384.
 -- 住宅地に点在する営み。
-The bustling activity throughout the residential district.
+The scattered pockets of activity in the residential district.
 }
 [sha:d296fb706eae90a818b55f2b2664c92f76d0bef1]{
 -- Page 190, Offset 31385.
 -- 国道を走る自動車のエンジン。
-The engines of running vehicles on the road.
+The drone of car engines racing down the road.
 }
 [sha:46867936f4e3f743e0e1fd8fcbed52d99e218e58]{
 -- Page 190, Offset 31386.
 -- 庭でざわめき合う木々の葉。
-The leaves of trees rustling against each other in the garden.
+The rustle of leaves in the garden.
 }
 [sha:f0bb61419590672e9521316dd73eddb90a7076e8]{
 -- Page 190, Offset 31387.
 -- そういった細やかな音が、今の俺にはよく聞こえる。
-As of now, I can clearly hear all these subtle sounds.
+Right now, I can hear all of these subtle sounds loud and clear.
 }
 [sha:2af2401da50abfb572f84635d9c7de603a82137f]{
 -- Page 191, Offset 31388.
 -- 「―――それにしても」
-"―――But even so..."
+"Still, despite all those sounds―――"
 }
 [sha:caa7b47df8eb3f781ed759b6c0449d862381678c]{
 -- Page 191, Offset 31389.
 -- ベッドの上で時間を数える。
-I count the hours from the top of my bed.
+Lying in bed, I count the hours pass.
```

Comment:
Something about the second clause is off for me with 'pass'. I think it might just be the 'that' which is missing (obviously, sometimes it's fine if it's missing, but here it felt weird)

>...I count the hours that pass